src/vmm/src/arch/aarch64/vstate/mod.rs

```diff
@@ -3,3 +3,7 @@
 
 /// Architecture specific KVM-related code
 pub mod kvm;
+/// Architecture specific vCPU code
+pub mod vcpu;
```

Comment:
mh, now we have two vcpu modules in `arch::aarch64`. Maybe we can merge them? Lots of the functions in the pre-existing arch::aarch64::vcpu module look like they should really be methods of `struct Vcpu` (am also happy to do this myself in a follow up tbh)

Reply:
+1